app/controllers/neighborhoods_controller.rb

```diff
@@ -0,0 +1,4 @@
+class NeighborhoodsController < ApplicationController
+	def new
```

Comment:
nit: people may want to fight with you over tabs vs. spaces 🤣

Reply:
Oh!  I normally use spaces...  (Is that the right answer?) But I edited these in Vim, since it was such a tiny change, and I must have hit tab by mistake.

Reply:
I usually use spaces, but that doesn't necessarily make it right